claat/cmd/export.go

```diff
@@ -45,7 +45,9 @@ type CmdExportOptions struct {
 }
 
 // CmdExport is the "claat export ..." subcommand.
-func CmdExport(opts CmdExportOptions) {
+// It returns a process exit code.
+func CmdExport(opts CmdExportOptions) int {
+	exitCode := 0
```

Comment:
nit: go style says to initialize empty values with `var` (`var exitCode int`), but this is also correct.